### PR TITLE
[Doc]Add namespace subscription-expiration-time operation doc on web page.

### LINKED ISSUE #9599: Admin CLI: namespaces set-subscription-expiration-time and get-subscription-expiration-time operations are not documented on web page
**Describe the bug**

**To Reproduce**
1. Downloaded Pulsar 2.7.0 and install it (`gunzip` and `tar -xvf`)
2. 
```
$ cd apache-pulsar-2.7.0/bin
$ ./pulsar-admin namespaces | grep expi
Warning: Nashorn engine is planned to be removed from a future JDK release
    get-subscription-expiration-time      Get subscription expiration time for 
      Usage: get-subscription-expiration-time tenant/namespace
    set-subscription-expiration-time      Set subscription expiration time for 
      Usage: set-subscription-expiration-time [options] tenant/namespace
            Subscription expiration time in minutes
```
No such documentation here --> https://pulsar.apache.org/docs/en/pulsar-admin/

**Expected behavior**
Documentation for `set-subscription-expiration-time` and `get-subscription-expiration-time` are present here
 https://pulsar.apache.org/docs/en/pulsar-admin/

**Additional context**

### PR DESCRIPTION
**Pull request by RobertIndie**
Fixes #9599 

### Motivation

Namespaces set-subscription-expiration-time and get-subscription-expiration-time operations are not documented on web page

### Modifications

Add namespace subscription-expiration-time operation doc on web page.

### CODE REVIEW

File: site2/website/versioned_docs/version-2.6.0/reference-pulsar-admin.md [hunk 1]
```diff
@@ -1351,6 +1353,27 @@ Usage
 $ pulsar-admin namespaces get-subscription-dispatch-rate tenant/namespace
 ```
 
+### `set-subscription-expiration-time`
+Set subscription expiration time in minutes for a namespace
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.6.0/reference-pulsar-admin.md:1357]:
```suggestion
Set subscription expiration time for a namespace (in minutes).
```

File: site2/website/versioned_docs/version-2.6.0/reference-pulsar-admin.md [hunk 2]
```diff
@@ -1351,6 +1353,27 @@ Usage
...
+$ pulsar-admin namespaces set-subscription-expiration-time tenant/namespace options
+```
+
+Options
+|Flag|Description|Default|
+|----|---|---|
+|`-t`, `--time`|Subscription expiration time in minutes|0|
+
+### `get-subscription-expiration-time`
+Get subscription expiration time for a namespace
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.6.0/reference-pulsar-admin.md:1370]:
```suggestion
Get subscription expiration time for a namespace (in minutes).
```

Is it correct?

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. Do not forget to correct docs for all related versions.

**Review by codelipenghui — APPROVED** [review]
Looks good to me, please address @Anonymitaet 's comments.

**Review by Huanli-Meng — APPROVED** [review]
Overall looks good to me. Please update the doc based on @Anonymitaet 's comments.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**